src/app/views/start/start.component.html

```diff
@@ -16,13 +16,10 @@ <h1>Plugin Terra Basic</h1>
             <hr />
             <h2>{{ 'startNow' | translate: lang }}</h2>
             <br />
-            <div class="center">
-                <terra-button
-                    [inputCaption]="'startHere' | translate: lang"
-                    (outputClicked)="redirectToDevelopersTutorial()"
-                    [inputTooltipText]="'startHere' | translate: lang"
-                >
-                </terra-button>
+            <div class="center" [tcTooltip]="(tooltipKey | translate: lang) ?? ''">
```

Comment:
Warum musst du das so komisch angeben? Kann dir translate Pipe etwa `null` zurückgeben?

Reply:
jo sie kann `string oder null` zurück geben

Reply:
Oh je.. Das wird dann richtig nervig in terra...

Reply:
Dann sollten wir denke ich entweder eine Pipe schreiben, die wir dahinter ketten, und die null rausfiltert.
Oder wir "überschreiben" die translate pipe mit einer eigenen implementation, die aber auf der der richtigen translate pipe basiert.

Reply:
Oha okay 😄

Reply:
done